src/challenges/counting_duplicate.js

```diff
@@ -0,0 +1,20 @@
+function duplicateCount(text) {
+  text = text.toLowerCase()
+  let charCounter = {}
+  let total = 0
+  for (const x of text) {
+    if (x in charCounter) {
+      charCounter[x] += 1
+    } else {
+      charCounter[x] = 1
+    }
+  }
+  for (const x in charCounter) {
+    if (charCounter[x] >= 2) {
+      total += 1
+    }
+  }
```

Comment:
Same, for practice always try to avoid for loops.

try a obj.keys().reduce() here.